### DISCUSSION TITLE
Shadow Updates master list

### DISCUSSION THREAD

**Discussion by DarkBum** [Roadhog360/Et-Futurum-Requiem#59] [Improvements]
- TNT drops 100% of destroyed blocks
- Iron Golems get cracked when hurt
- Item Frames have 8 rotations, instead of 4
- [I'll expand this if I remember more of these]

**Comment by DarkBum** [comment]
- The workings of the Slime Block are just a tiny tad off [The mod "Back in Slime" might help.
- The Banner received another pattern, the Globe [I know, it has not crafting recipe, so this could either be added with a custom recipe, or when you add the Loom]
- All suspicious stews can now be eaten at full hunger
- Coarse dirt can be converted into Dirt Path [on that occasion, you could make the Dirt Path a non-full block, like in UpToDate ;) ]
- Rabbit stew can be fed to Wolves
- Rabbits should jump away and flee from players and hostile mobs that get near them
- Maybe add arms to the Armor Stands [I know, it's a Bedrock feature, but... eh...]
- A multitude of Advancements that should be remade in the Achievments tab :)
- A mutlitude of Enchantments that are missing [probably overlooked :3]
- Splashes? [more of a joke feature ;) ]
- Beacon beams also go through blocks that don't block all light [like Water, leaves, ice, etc.]
- There is a distinction between naturally grown Pumpkins and Carved Pumpkins.
- Dispensers place Pumpkins and Wither Skeleton Skulls on their respective bodies to spawn Golems and Withers.
- Dispensers also place command blocks with pre-configured commands.
- Buttons can be placed on the ground and ceiling.
- Item Frames aswell.
- Baby Zombies burn in the sun
- Redstone repeaters and comparators no longer emit light [seriously, why?]
- Activator rails shake minecards and dismount riders
- The top texture of Jack o' Lanterns [and Carved Pumpkin] rotates with their face
- Models of torches [all of them] have and angled head when placed on a wall [???]
- Heads, signs and beacons display the breaking animation
- All kinds of map shenanigans [seriously, holy fuck, how much these got altered]
- Signs and books use JSON text components as formatting [Yeah, I don't know...]
- Caverns on the surface of Deserts, Mushroom and Mesa biomes [the latter show Red Sandstone at the entrance]
- Copying a block in Creative Mode copies all their NBT data

**That's now mostly 1.8 stuff. Will update later versions**

**Comment by Roadhog360** [comment]
Pinned this

**Comment by Roadhog360** [comment]
Certain objects can fuel a furnace like boats (Needs more documentation)
Mobs in boats cannot despawn

**Comment by DarkBum** [comment]
- Potion of Leaping can be extended from 3:00 to 8:00 with Redstone
- A plethora of new statistics [https://minecraft.fandom.com/wiki/Java_Edition_1.8.2]
- Beacons beams ignore Bedrock in their path [Delirus has added this feature for his Nether Beacon]
- Crafting a Carrot on a Stick removes all Enchantments from the Fishing Rod
- Dispensers can place down command blocks with commands already inserted [I know, that is reversed later, but it's such a cool feature!]
- Up to 16 layers of Banners are rendered

**Now that's the rest of what I found interesting about 1.8.X**

**Comment by Roadhog360** [comment]
Unpinned to make room for a different future pinned thread (sorry!)
